torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -1100,6 +1100,49 @@ void InsertObserversHelper::fillBoundaryValueMap(
   }
 }
 
+void makeAppendNonInplace(std::shared_ptr<Graph>& graph) {
+  std::string append_pattern = R"IR(
```

Comment:
Can you elaborate more on why directly supporting append is not possible?

Reply:
All current ops including inplace ops assumes that the output will be consumed by the following ops. To break this assumption, we'll need to introduce substantial changes/hacks, I don't think it's worth the effort

Reply:
btw, if there are perf problems we can easily add a pass to change the add to append in the end